ansible/l1_port_mapper.py

```diff
@@ -0,0 +1,148 @@
+import argparse
+import csv
+import json
+from typing import Dict, List, Tuple
+import os
+
+
+def get_abs_path(filename):
+    csv_dir = os.environ.get("PYTHONPATH", "")
+    return os.path.join(csv_dir, filename)
+
+
+def read_csv(file_path: str) -> List[Dict[str, str]]:
```

Comment:
Hi @developfast , could you please fill in more background/context in the PR description.
Also a sample of csv file would be helpful.